Library/Homebrew/cmd/config.rb

```diff
@@ -60,7 +60,7 @@ def origin
   def describe_path(path)
     return "N/A" if path.nil?
     realpath = path.realpath
-    if realpath == path then path else "#{path} => #{realpath}" end
+    realpath == path ? path : "#{path} => #{realpath}"
```

Comment:
Sure, if you prefer it that way on readability/etc, happy to pull it onto multiple lines instead.